doc/api_assets/style.css

```diff
@@ -1,5 +1,6 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  -webkit-overflow-scrolling: touch;
```

Comment:
Yes that's correct web pages, by default, don't have the same inertia apps have.